licenses/xml/gfdl.xml

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0"?>
+<article version="5.2" xml:id="gfdl" xml:lang="en"
+	 xmlns="http://docbook.org/ns/docbook"
+	 xmlns:xlink="http://www.w3.org/1999/xlink">
+  <info>
+    <title>GFDL -- GNU Free Documentation License</title>
```

Comment:
Do we need to add  `GFDL  --  `? In the former appendices, we just had `GNU Free Documentation License`. If at all, I would rather add the acronym at the end: `GNU Free Documentation License (GFDL)`. At least in PDF output, the current title looks a bit odd:
![Screenshot_20240118_131830](https://github.com/SUSE/doc-unversioned/assets/1875958/806630b4-ad52-46fa-b2db-170ced3d7f1a)

Reply:
Good points! I've implemented that in commit 4a3e5de.